### PR TITLE
Fix/styles review

### PR DESCRIPTION
**Pull request by BC-Mikey**
Styles updates & fixes

### CODE REVIEW

File: css/bcgs_ford_controls.css [hunk 1]
```diff
@@ -41,43 +44,16 @@
...
-        top: 14px !important;
-    }
-    
-    .bcc-filmstrip-opened .bcc-player-button.bcc-filmstrip-control:before, .bcc-player-button.bcc-filmstrip-control:hover:before {
-        content: '';
-    }
-
-.bcc-player-button.bcc-social-control {
+.bcc-social-control {
+    height: 18px !important;
```
Comment (bceacuna) [inline css/bcgs_ford_controls.css]: Please also update the class name here
https://github.com/BrightcovePS/FordPlayer/blob/dev/lib/bcgs_ford_modal.js#L80

So after you merge we can bring back the share icon on smart phones.

**Comment by bceacuna** [comment]
Looks good. Merge away after updating bcgs_ford_modal.js

**Comment by bceacuna** [comment]
The svg image was added to the detail panel but it is white (over a white
background).  Can you please fix this.



bcc-social-control

Thanks

On Fri, Jul 10, 2015 at 4:26 PM, Mikey Carreiro <notifications@github.com>
wrote:

> Merged #55 <https://github.com/BrightcovePS/FordPlayer/pull/55>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BrightcovePS/FordPlayer/pull/55#event-353288814>.
>



-- 


*Erick Acuna** |*  Principal Software Engineer
Brightcove, Inc. http://www.brightcove.com/  <http://www.brightcove.com/>
290 Congress Street, 4th Floor | Boston, MA 02110
P: 617.3950494

**Comment by BC-Mikey** [comment]
Done:

https://github.com/BrightcovePS/FordPlayer/pull/56

On Fri, Jul 10, 2015 at 1:39 PM, bceacuna <notifications@github.com> wrote:

> The svg image was added to the detail panel but it is white (over a white
> background). Can you please fix this.
>
>
>
> bcc-social-control
>
> Thanks
>
> On Fri, Jul 10, 2015 at 4:26 PM, Mikey Carreiro <notifications@github.com>
> wrote:
>
> > Merged #55 <https://github.com/BrightcovePS/FordPlayer/pull/55>.
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/BrightcovePS/FordPlayer/pull/55#event-353288814>.
> >
>
>
>
> --
>
>
> *Erick Acuna** |* Principal Software Engineer
> Brightcove, Inc. http://www.brightcove.com/ <http://www.brightcove.com/>
> 290 Congress Street, 4th Floor | Boston, MA 02110
> P: 617.3950494
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/BrightcovePS/FordPlayer/pull/55#issuecomment-120516479>
> .
>



-- 

Michael Carreiro

Solutions Engineer, Brightcove Consulting

Brightcove Inc. <http://www.brightcove.com/>

**Pull request merged**